### PR TITLE
feat: added my-cart page

### PR DESCRIPTION
**Pull request by Radheshyam1234**

### CODE REVIEW

**Review by lannisterrr — APPROVED** [review]
Code looks structure and best practices followed.

File: my-cart.html [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+          <span class="badge-container">
+            <i class="fas fa-shopping-cart">
+              <span class="badge-icon badge-pink">1</span></i
+            >
+          </span>
+        </a>
+      </div>
+    </header>
+
+    <div class="spacer-3rem"></div>
```
Comment (bhanuprakashj) [inline my-cart.html:46]: just a suggestion better to avoid empty elements as much as possible

**Review by Tanay-J — APPROVED** [review]
Clean code. I think you can merge it.

File: my-cart.html [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+          <div class="w-70">Discount</div>
+          <div class="w-30">-₹ 1999</div>
+        </div>
+
+        <div class="row text-semibold">
+          <div class="w-70">Delivery Charges</div>
+          <div class="w-30">₹ 499</div>
+        </div>
+
+        <hr />
```
Comment (bhanuprakashj) [inline my-cart.html:237]: suggesstion: instead of hr use border property for styling which is more flexible

**Review by bhanuprakashj — COMMENTED** [review]
rest all looks good to me

File: my-cart.html (hunk 2 above)
Comment (Radheshyam1234) [reply my-cart.html:237]: Ok i will do

**Pull request merged**